src/Agent/NewRelic/Agent/Core/Config/Configuration.xsd

```diff
@@ -1655,6 +1655,15 @@
                                             </xs:documentation>
                                         </xs:annotation>
                                     </xs:attribute>
+
+                                    <xs:attribute name="logLevelDenyList" type="xs:string" use="optional">
+                                        <xs:annotation>
+                                            <xs:documentation>
+                                                A comma-separated, case-insensitive, list of log levels that should be ignored and not sent up to New Relic.
```

Comment:
Should this call out that the level name used by their logging framework should be used?